types/owner.go

```diff
@@ -25,7 +27,52 @@ type WorkspaceOwner struct {
 	RBACRoles []WorkspaceOwnerRBACRole `json:"rbac_roles"`
 }
 
+func (o *WorkspaceOwner) ToCtyValue() (cty.Value, error) {
+	convertedGroups, err := gocty.ToCtyValue(o.Groups, cty.List(cty.String))
+	if err != nil {
+		return cty.Value{}, err
+	}
+
+	roleValues := make([]cty.Value, 0, len(o.RBACRoles))
+	for _, role := range o.RBACRoles {
+		roleValue, err := role.ToCtyValue()
+		if err != nil {
+			return cty.Value{}, err
+		}
+		roleValues = append(roleValues, roleValue)
+	}
+	var convertedRoles cty.Value = cty.ListValEmpty(WorkspaceOwnerRBACRole{}.CtyType())
+	if len(roleValues) > 0 {
+		convertedRoles = cty.ListVal(roleValues)
+	}
```

Comment:
Can you not just do this? It should cover the empty case, since you define the list type.

```suggestion
	roleValues, err := gocty.ToCtyValue(o.RBACRoles, cty.List(WorkspaceOwnerRBACRole{}.CtyType()))
	if err != nil {
		return cty.Value{}, err
	}
```

Reply:
no, because `uuid.UUID`

Reply:
Daaaang